tests/conftest.py

```diff
@@ -105,13 +106,13 @@ def compileSolidity(relativeFilePath):
                 }
             },
             'settings': {
-                'remappings': [ 'ROOT=/augur-core/src' ]
+                'remappings': [ 'ROOT=%s' % resolveRelativePath("../../augur-core/src") ]
             },
             'outputSelection': {
                 '*': [ 'metadata', 'evm.bytecode', 'evm.sourceMap' ]
             }
         }
-        return compile_standard(compilerParameter, allow_paths="/augur-core/src")['contracts'][filename][contractName]
+        return compile_standard(compilerParameter, allow_paths=resolveRelativePath("../../augur-core/src"))['contracts'][filename][contractName]
```

Comment:
Remove `../augur-core`.  That will _only_ work if someone clones `augur-core` GitHub project into a folder named `augur-core`, which shouldn't be assumed.